test/case/ietf_interfaces/igmp_vlan/test.py

```diff
@@ -46,8 +46,10 @@
 
 
     with test.step("Configure device"):
-        dut1.put_config_dict("ietf-interfaces",
-                             {
+        dut1.put_config_dicts(
+            {
+                "ietf-interfaces":
+                               {
```

Comment:
```suggestion
                "ietf-interfaces": {
```
...then remove lots of superfluous indentation below